pages/exercises/[lessonSlug].tsx

```diff
@@ -23,9 +24,21 @@ const Exercises: React.FC<QueryDataProps<GetAppQuery>> = ({ queryData }) => {
   if (!currentLesson)
     return <Error code={StatusCode.NOT_FOUND} message="Lesson not found" />
 
+  const tabs = [
+    ...(currentLesson.docUrl
+      ? [{ text: 'lessons', url: currentLesson.docUrl }]
+      : []),
```

Comment:
This code doesn't look very good. This makes it so if the current lesson doesn't have a docUrl, it will not show a LESSONS NavCard link. I wasn't sure if there was a better way to do this or if anyone has any suggestions, feel free to comment ideas.